static/_redirects

```diff
@@ -369,3 +369,8 @@
 /subnets/README /learn/avalanche/subnets-overview 301
 /overview/projects /learn/projects 301
 /subnets /learn/avalanche/subnets-overview 301
+
+#05/24/2023
+/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
+/avalanchego-release-notes#v1912-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12 301
+/avalanchego-release-notes#v1714-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.7.14 301
```

Comment:
Please manually verify the first part is a valid page on https://docs.avax.network/. These two are not valid.

Reply:
https://docs.avax.network/apis/avalanchego/avalanchego-release-notes is the currently valid page that is being removed, which is why I originally had this line to redirect:
`/apis/avalanchego/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301`

I am a bit confused

Reply:
I removed /apis/avalanchego/avalanchego-release-notes/, but there is still /apis/avalanchego/avalanchego-release-notes which should be kept.

Reply:
Understood. Verified each link manually and corrected the redirects. Thank you for the explanation/patience.